apps/cms/pages/templates/a4_candy_cms_pages/blocks/col_cta_block.html

```diff
@@ -10,10 +10,11 @@
             <div class="block-col_cta-content">
                 <a href="{{ column.link }}" class="btn btn--full btn--transparent">{{ column.link_text }}</a>
             </div>
-            {% else %}
+            {% elif not column.body and column.link %}
             <div class="block-col_cta-btn">
                 <a href="{{ column.link }}" class="btn btn--full btn--transparent">{{ column.link_text }}</a>
             </div>
+            {% else %}
```

Comment:
does this case exist? `not column.body and not column.link`

Reply:
sorry forgot to add issue to commit https://github.com/liqd/adhocracy-plus/issues/2002